cmd/scouting-backend/main.go

```diff
@@ -16,8 +16,8 @@ import (
 // PG_PORT: postgres port
```

Comment:
We're going to need to make sure this stays in sync with the readme

Reply:
Yeah IK it's kind of annoying we might just remove it.